### ISSUE TITLE
Results saved to our sorry-cypress dashboard but the CI run failed due to "We encountered an unexpected error talking to our servers."

### ISSUE THREAD

**Issue by sdickerson216** [sorry-cypress/sorry-cypress#206]
## Summary

This is a weird one because the error message seems to be coming from a Cypress server, but the api url is pointing to our own sorry-cypress-director service on Heroku. After CircleCI completes a test run, it will randomly report "We encountered an unexpected error talking to our servers." and fail the run. But, if I look at the results in our sorry-cypress dashboard, all of the results are there. We have 3 containers running in parallel, and usually only 1 or 2 of them report this error.

I was wondering if anyone else has seen this issue. Our script updates the api url after Cypress is installed and before the tests are run:
sed -i -e 's|api_url:.*$|api_url: "https://th-cypress-director.herokuapp.com/"|g' /*/.cache/Cypress/*/Cypress/resources/app/packages/server/config/app.yml

Do we need to run any other command to update the api url in a different file?

## How to reproduce
This seems to only occur when Cypress servers are having an issue, but I didn't think Cypress would be communicating with them in this setup. This error was happening a lot on 11/2/20, but it seems Cypress has corrected the server issue.

## Environment

- sorry-cypress version: `currently v1.0.0-beta.9 but I was getting it in previous versions`
- cypress version: `5.3.0`
- platform: `docker on CircleCI, parallelism: 3`
- service: `all`

**Comment by sdickerson216** [comment]
I spoke too soon. Two out of the three parallel containers just reported this error in our last CI run.

![Screen Shot 2020-11-04 at 3 34 19 PM](https://user-images.githubusercontent.com/36859415/98175218-65278480-1eb3-11eb-95e7-46fe4e3b63c1.png)

**Comment by agoldis** [comment]
Do you happen to have the full error message after `StatusCodeError: 503`? It looks like heroku error message - i.e. sorry-cypress response. 

Interesting, I didn't know cypress agent tries to talk to the original servers.

**Comment by sdickerson216** [comment]
Hi Andrew, thanks for responding! You are correct -- the error is coming from my Heroku app and not from the Cypress servers like I thought. When I upgraded my sorry-cypress services I also moved the db from mLab to Atlas because mLab will not be supported on Heroku after 11/10/20. 

It seems that the free tier of Atlas is painfully slow, which is causing this issue plus the other timeout issue I was running into. After I switched the db to a paid add-on for ObjectRocket, the sorry-cypress services started working a lot more reliably.
